src/client/components/atoms/burgerButton/index.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import { useContext, useState } from 'react';
+import { Link } from 'react-router-dom';
+import StateContext from '../../../utils/stateContext';
+import ExpandMenu from '../expand';
+import './styles.scss';
+
+const BurgerButton = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const { user } = useContext(StateContext);
+
+  const handleToggleMenu = () => {
+    setIsOpen(!isOpen);
+  };
+  return (
+    <div>
+      <button className={classNames(`burger ${isOpen ? 'open' : ''}`)} onClick={handleToggleMenu}>
+        <span className="bar"></span>
+        <span className="bar"></span>
+        <span className="bar"></span>
+      </button>
+      <ExpandMenu isOpen={isOpen}>
+        {' '}
```

Comment:
we should remove this, if you need to make some space or move elements around, doit via css